services/nginx-proxy/docker-compose.yml

```diff
@@ -22,16 +22,17 @@ services:
           - draft-router.dev.gov.uk
           - draft-static.dev.gov.uk
           - government-frontend.dev.gov.uk
-          - govuk-developer-docs.dev.gov.uk
           - govuk_publishing_components.dev.gov.uk
```

Comment:
Not exactly in this PR, but I believe this needs updating to use '-' instead of '_'.